packages/manager/modules/pci/src/projects/project/storages/databases/database/databases/create-database/create-database.html

```diff
@@ -1,25 +1,37 @@
-<oui-modal
-    data-on-dismiss="$ctrl.goBack()"
-    data-primary-label="{{:: 'pci_databases_databases_create_database_confirm' | translate}}"
-    data-primary-disabled="!$ctrl.databaseName"
-    data-primary-action="$ctrl.addDatabase()"
-    data-secondary-action="$ctrl.cancel()"
-    data-secondary-label="{{:: 'pci_databases_databases_create_database_cancel' | translate}}"
-    data-loading="$ctrl.processing"
-    data-heading="{{:: 'pci_databases_databases_create_database' | translate}}"
->
-    <p data-translate="pci_databases_databases_create_database_description"></p>
-    <oui-field
-        data-label="{{:: 'pci_databases_databases_create_database_label' | translate }}"
+<form name="addDatabase">
```

Comment:
if you are adding a form i believe you should use an ng-submit

Reply:
Fixed: https://github.com/ovh/manager/pull/5620/commits/36a994f957d25cc6adef8315f7e344b098d4286a